repl/tests/print.glu

```diff
@@ -0,0 +1,2 @@
+let io = import! "std/io.glu"
+io.println "123"
```

Comment:
Missing trailing new line on this file 😆

Reply:
😞 I thought I ran with editorconfig to ensure this would never be a problem... Maybe the something broke in a recent update.